addons/repair/models/repair.py

```diff
@@ -395,6 +395,7 @@ def _create_invoices(self, group=False):
                     raise UserError(_('No product defined on fees.'))
 
                 account = fee.product_id.product_tmpl_id._get_product_accounts()['income']
+                account = self.env['account.fiscal.position'].browse(fp_id).map_account(account)
```

Comment:
we could use `get_product_accounts` instead

Reply:
Sorry if I didn't elaborate further, I was suggesting that `get_product_accounts` already implement the account mapping, so we can call it with the fiscal position
```python
fp = self.env['account.fiscal.position'].browse(fp_id)
account = fee.product_id.product_tmpl_id.get_product_accounts(fiscal_pos=fp)['income']
```

Reply:
Understood, it's fixed :ok_hand: